v2/internal/testcommon/vcr/redact.go

```diff
@@ -16,6 +16,22 @@ import (
 	"github.com/Azure/azure-service-operator/v2/internal/testcommon/creds"
 )
 
+type Redactor struct {
+	azureIDs   creds.AzureIDs
```

Comment:
Not for this PR, but we might want to get rid of this - and just use the value in `NewRedactor` to preinitialize values in `redactions`.

Reply:
Nice! I've taken this too